assets/generated_samples_list.json

```diff
@@ -992,7 +1022,7 @@
       "OGC",
       "WMS",
       "web map service",
-      "ArcGISMapViewController.identifyLayer",
+      "ArcGISMapViewController.identifyLayerAsync",
```

Comment:
This was fixed in some other PR. Maybe do a merge to make sure the PR's aren't cancelling each other out. "identifyLayer" is what we want.